includes/SiteGen/SiteGen.php

```diff
@@ -565,6 +598,24 @@ function ( $key ) use ( $menu_patterns_slugs ) {
 			}
 		}
 
+		// if contact page then pick from contact page templates
+		// also make sure the jetpack plugin is installed and active
+		// then activate "contact-form" module since some templates use jetpack forms
+		if ( 'contact' === $page ) {
+			$contact_page_templates = self::get_templates_for_category( $page );
+			// templates fetched successfully
+			if ( ! isset( $contact_page_templates['error'] ) ) {
+				$random_contact_page_template_slug = array_rand( $contact_page_templates, 1 );
+				$contact_page_content              = $contact_page_templates[ $random_contact_page_template_slug ]['content'];
+
+				// install and activate the Jetpack plugin and enable the "contact-form" module
+				if ( Plugins::install( 'jetpack' ) && Plugins::activate_jetpack_module( 'contact-form' ) ) {
```

Comment:
Can we check if Jetpack is already installed and active before making this call? We should be able to use this method: https://github.com/newfold-labs/wp-module-installer/blob/2f7b1a2d128703119bc5090e93e9cdd3045044b4/includes/Services/PluginInstaller.php#L326

Reply:
The install function also checks whether its installed https://github.com/newfold-labs/wp-module-installer/blob/2f7b1a2d128703119bc5090e93e9cdd3045044b4/includes/Services/PluginInstaller.php#L87 and if its active https://github.com/newfold-labs/wp-module-installer/blob/2f7b1a2d128703119bc5090e93e9cdd3045044b4/includes/Services/PluginInstaller.php#L97 .. so thought of killing 2 birds with one stone  ... lets discuss the correct approach.